packages/webpack-loader/src/__tests__/compiled-loader.test.tsx

```diff
@@ -15,7 +15,7 @@ describe.each<'development' | 'production'>(['development', 'production'])(
         mode,
       });
 
-    it('does not transform files that do not contain @compiled/react', async () => {
+    it.concurrent('does not transform files that do not contain @compiled/react', async () => {
```

Comment:
Seemed to shave off some time overall so I added these in. It means it runs each test at the same time.